Action_inputs.md

```diff
@@ -0,0 +1,135 @@
+## Action inputs
+
+### Inputs specific to set cluster context
+<table>
+  <thead>
+    <tr>
+      <th>Action inputs</th>
+      <th>Description</th>
```

Comment:
Please have a septate column for 'required'/'optional'

Reply:
done